doc/mainpage.dox

```diff
@@ -111,15 +109,6 @@ To build only the documentation:
 
 See the unit tests in `tests` and the examples in `examples`
 
-## Plans
-
-- Improve error reporting so exceptions have useful text and no text is printed to stderr.
-
-Longer term desires:
-- Support AST regions
-- Add support for building with cmake.
-- Add doc strings to python wrapped objects.
```

Comment:
Yes. At this point it's close enough to done that I don't feel the need to talk about the future.